hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/systemcontracts/ExchangeRateSystemContractTest.java

```diff
@@ -118,6 +122,17 @@ void inputCannotUnderflow() {
         assertThat(result.result().getHaltReason().get()).isEqualTo(ExceptionalHaltReason.INVALID_OPERATION);
     }
 
+    @Test
+    void valueShouldNotBeSentToThePrecompile() {
+        given(frame.getValue()).willReturn(Wei.MAX_WEI);
+
+        final var zeroInput = tinycentsInput(0);
+        final var result = subject.computeFully(zeroInput, frame);
+
+        assertThat(result.output()).isEqualTo(Bytes.EMPTY);
+        assertThat(result.result().getHaltReason().get()).isEqualTo(ExceptionalHaltReason.INVALID_OPERATION);
```

Comment:
Shouldn't we test for the new `INVALID_FEE_SUBMITTED` case?

Reply:
A new version available. Thank you.